### PR TITLE
STREAMS-2248 docs(upgrade): update comment regarding authorization flow…

### PR DESCRIPTION
**Pull request by rbissey-axway**
… for publisher-http-poller and subscriber-webhook.

Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Remove token limitation for Publisher-http-poller and Subscriber-webhook.
https://deploy-preview-136--streams-open-docs.netlify.app/docs/publishers/publisher-http-poller/
https://deploy-preview-136--streams-open-docs.netlify.app/docs/subscribers/subscriber-webhook/

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

### CODE REVIEW

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The Webhook subscriber can post data to an API that is secured with [OAuth2](htt
 
 The OAuth2 authorization workflow is implemented with the following limitations:
 
-* The OAuth2 authorization workflow is initiated on the authorization server URL each time data is posted. Refresh token mechanism is not implemented.
+* The OAuth2 authorization workflow is initiated on the authorization server URL for the first polling and the retrieved access token is reused for following requests until it expires. Refresh token mechanism is not implemented.
```
Comment (andreamussap) [inline content/en/docs/subscribers/subscriber-webhook.md:54]: ...for the first poll**ing,** and the retrieved access token...

Wouldn't be more clear to put "Refresh token mechanism is not implemented" as new bullet point?

Comment (rbissey-axway) [reply content/en/docs/subscribers/subscriber-webhook.md]: Thanks @andreamussap, that actually looks better to me. 
I update my commit and wait for @guillaumelamirand 's approval.

**Review by guillaumelamirand — APPROVED** [review]

**Comment by epiasentin-axway** [comment]
@andreamussap If it is OK for you, you can merge this PR.

File: content/en/docs/publishers/publisher-sfdc.md [hunk 2]
```diff
@@ -3,61 +3,63 @@ title: Salesforce Publisher
 linkTitle: Salesforce Publisher
 weight: 4
 date: 2020-07-02
-description: Learn how to configure a topic associated to a Salesforce Publisher.
+description: The Salesforce Publisher provides the capability to capture changes from Salesforce.com via Salesforce Streaming API PushTopics or Salesforce Platform Events. Learn how to configure a topic associated to a Salesforce Publisher.
 ---
 
 {{< alert title="Beta feature" color="warning" >}}
-Salesforce Publisher is still experimental, and will be enhanced in future releases.
+Salesforce Publisher is a beta feature.
```
Comment (guillaumelamirand) [inline content/en/docs/publishers/publisher-sfdc.md:10]: Kakfa publisher must be aligned with this changes.

File: content/en/docs/publishers/publisher-sfdc.md [hunk 3]
```diff
@@ -3,61 +3,63 @@ title: Salesforce Publisher
...
 {{< /alert >}}
 
-## Salesforce Publisher
+PushTopics provide the ability to subscribe to events (changes in a record), for example, create a record, delete a record, and so on, whereas Platform Events allow Salesforce users to define their own publish/subscribe events. After integrated with Streams, Salesforce events can then be broadcast by any of Streams [subscribers](/docs/subscribers).
```
Comment (guillaumelamirand) [inline content/en/docs/publishers/publisher-sfdc.md:13]: I would prefer to keep the reference to Salesforce Objects (SObjects). 

I suggest 

PushTopics provide the ability to subscribe to change related to Salesforce Objects (SObjects) whereas Platform Events allow Salesforce users to define their own publish/subscribe events. After integrated with Streams, Salesforce events can then be broadcast by any of Streams [subscribers](https://docs.axway.com/bundle/streams-open-docs/page/docs/subscribers/index.html).

**Pull request merged**